IPython/utils/module_paths.py

```diff
@@ -26,25 +26,16 @@
 from __future__ import print_function
 
 # Stdlib imports
-import imp
 import os
-
-# Third-party imports
-
-# Our own imports
-
+import sys
+import warnings
 
 #-----------------------------------------------------------------------------
-# Globals and constants
+# Classes and functions
 #-----------------------------------------------------------------------------
```

Comment:
We're mostly removing these blocks entirely as we work on the files. Not important, though.